tests/schemas/invalid/test_invalid_progress_value_source_block_in_past_repeating_section.json

```diff
@@ -0,0 +1,352 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "language": "en",
+  "schema_version": "0.0.1",
+  "survey_id": "139",
+  "theme": "default",
+  "title": "Confirmation Question Test",
```

Comment:
Titles and descriptions for the test schemas will need updating 👍

Reply:
Addressed